letsencrypt/cli.py

```diff
@@ -56,7 +56,7 @@
 the cert. Major SUBCOMMANDS are:
 
   (default) run        Obtain & install a cert in your current webserver
-  auth                 Authenticate & obtain cert, but do not install it
+  certonly             Obtain cert, but do not install it (aka "auth")
```

Comment:
Yes, for at least a while, because a lot of people have heard about "auth" and should experience some continuity.